### PR TITLE
Base styling

### PR DESCRIPTION
**Pull request by wendsli**
- built base CSS styling in application.css
- updated page title and header with desired font and formatting

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -10,8 +10,41 @@
...
+}
+
+.sticky {
+    width: 100%;
+}
+
+.content {
+    margin: 2em;
+    margin-top: 3em;
+}
```
Comment (dovakeen118) [inline app/assets/stylesheets/application.css:50]: yay styling! these should be removed from this manifest file and put in their own stylesheet like `main.css`

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -1,45 +1,52 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>Podcastr</title>
+  <title>ρodcastr</title>
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   <%= stylesheet_link_tag 'application', media: 'all' %>
   <%= csrf_meta_tags %>
+  <link href="https://fonts.googleapis.com/css?family=Play&display=swap" rel="stylesheet">
```
Comment (dovakeen118) [inline app/views/layouts/application.html.erb:8]: this should be imported from the stylesheet

File: app/assets/stylesheets/application.css (hunk 1 above)
Comment (wendsli) [reply app/assets/stylesheets/application.css]: Moved into `main.scss` per suggestion

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (wendsli) [reply app/views/layouts/application.html.erb]: Moved to importing font into `application.css`

**Pull request merged**